includes/definitions/efficientip.yaml

```diff
@@ -0,0 +1,13 @@
+os: efficientip
```

Comment:
efficientip is the company isn't it? It should be named after the OS that the devices run. If the OS doesn't have a name then use solidserver.